x-pack/plugins/apm/server/lib/helpers/transaction_query.ts

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
`transaction_query.ts` is perhaps a bit too generic, since we also have [`get_transaction`](https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/lib/transactions/get_transaction.js) that doesn't use this.

Reply:
I'll call it transaction_group_query